lib/compiler/src/beam_ssa_alias.erl

```diff
@@ -831,17 +831,17 @@ aa_get_status_by_type(Type, StatusByType) ->
 aa_alias_surviving_args(Args, Call, SS, Anno, AAS) ->
     KillSet = aa_killset_for_instr(Call, AAS),
     ArgTypes = maps:get(arg_types, Anno, #{}),
-    aa_alias_surviving_args1(Args, 0, SS, ArgTypes, KillSet).
+    aa_alias_surviving_args1(Args, SS, ArgTypes, KillSet).
```

Comment:
Drop `ArgTypes` too, it's not used, and when that's done, we can drop `Anno` too. Then we can drop `Anno` from `aa_phi/5`...

Before the introduction of the new sharing state database, the index was used to look up argument types to avoid wholesale aliasing of tuples when extracted fields were given to a call. With the new representation this is harmless, and as we're no longer annotating all instructions which use `aa_alias_surviving_args` with types, we can just ignore the types and annotations.